docs/examples/catch.md

```diff
@@ -0,0 +1,198 @@
+# Catch
+
```

Comment:
need to wrap this guy in a code block:

\```blocks
and
\``` after the end of the javascript

Reply:
Fixed.